### PR TITLE
Package liquidsoap.2.0.6

### PR DESCRIPTION
**Pull request by toots**
### `liquidsoap.2.0.6`
Swiss-army knife for multimedia streaming
Liquidsoap is a powerful and flexible language for describing your
streams. It offers a rich collection of operators that you can combine
at will, giving you more power than you need for creating or
transforming streams. But liquidsoap is still very light and easy to
use, in the Unix tradition of simple strong components working
together.

---
* Homepage: https://github.com/savonet/liquidsoap
* Source repo: git+https://github.com/savonet/liquidsoap.git
* Bug tracker: https://github.com/savonet/liquidsoap/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

**Comment by toots** [comment]
Hi! Any update on this one? Thanks!

File: packages/liquidsoap/liquidsoap.2.0.6/opam [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+homepage: "https://github.com/savonet/liquidsoap"
+bug-reports: "https://github.com/savonet/liquidsoap/issues"
+depends: [
+  "conf-autoconf" {dev}
+  "conf-automake" {dev}
+  "ocaml" {>= "4.12.0" & < "5.0.0"}
+  "camomile" {>= "1.0.0"}
+  "dtools" {>= "0.4.4"}
+  "duppy" {>= "0.9.1"}
+  "menhir" {>= "20180703" & <= "20211128"}
```
Comment (kit-ty-kate) [inline packages/liquidsoap/liquidsoap.2.0.6/opam:22]: Sorry for the delay, we seem to have forgotten about this PR. One question before I merge, is it still required?
```suggestion
  "menhir" {>= "20180703"}
```

Comment (toots) [reply packages/liquidsoap/liquidsoap.2.0.6/opam:22]: I think so, yes.

Comment (kit-ty-kate) [reply packages/liquidsoap/liquidsoap.2.0.6/opam:22]: I just tried with 20220210 and it compiled just fine locally

**Comment by toots** [comment]
Actually: https://github.com/savonet/liquidsoap/runs/7099558709

**Comment by toots** [comment]
Ok I see. It works on `opam` because the tarball ships with the compiled parser. I suppose it's fine to leave as is for now.

**Comment by toots** [comment]
We also have lifted this limitation on `2.1.x`, which is still autoconf. I think @smimram did some rework of the build system there.

**Comment by kit-ty-kate** [comment]
Ah I see, I’ve only looked at master branch. Well all green on this PR. Thanks a lot!

**Pull request merged**